src/components/LanguageMenu/LanguageMenu.tsx

```diff
@@ -34,7 +34,7 @@ export const LanguageMenu = ({ inline, onClick }: LanguageMenuProps) => {
   return inline ? (
     <ListItems onClick={onClick} />
   ) : (
-    <PopupList classNames="!bg-bg-gray-2">
+    <PopupList classNames="!bg-bg-gray-2 gap-y-2">
```

Comment:
Not really the part of this PR, but noticed that Popuplist is used in two places and both adds different gap than the component already odes, so maybe gap should be updated there.

And this background matches with the design, but it is weird that only language menu has gray background (other two menus nearby; MegaMenu and user menu does not)